sui_types/src/crypto.rs

```diff
@@ -485,10 +485,30 @@ impl PartialEq for AuthoritySignInfo {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
```

Comment:
Also, we should note that certified txs can carry more than quorum sigs (every authority might in theory sign), and wondering if we should put a comment here to clarify it.